topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -455,23 +506,24 @@ the two samples?
 {: .hands_on}
 
 
-To further explore the community structure, we can visualize it with dedicated tools such as Phinch:
+To further explore the community structure, we can visualize it with dedicated tools such as Phinch.
 
 > ### {% icon hands_on %} Hands-on: Visualization of the community structure with Phinch
 >
 > 1. **Make.biom** {% icon tool %} with the following parameters
->   - "shared" to Make.shared
->   - "constaxonomy" to taxonomy output from the first run of `classify.otu` (collection)
+>   - "shared" to `Make.shared` output
+>   - "constaxonomy" to taxonomy output from the first run of `Classify.otu` (collection)
+> 1. On "calculator" collection, click on "Visualize" and then "Phinch"
+> 
+>     > ### {% icon comment %} Comment
+>     >
+>     > If the link is not present on the user server (since it is not turned on by default), the alternative is to download the generated BIOM file and upload directly to Phinch server at [http://phinch.org](http://phinch.org).
```

Comment:
maybe "If this link is not present on your Galaxy, you can download.." instead of "the user server"?